### DISCUSSION TITLE
Промежуточный вариант ЕЖ3 для ЕЖ3, звонок на еж3 и 81-717МСК

### DISCUSSION THREAD

**Discussion by fixinit75** [metrostroi-repo/MetrostroiAddon#203] [General]
Но это требует манипуляций с моделями/текстурами пульта от ЕЖ3. Аналогично, 81-508Т может быть головным.
Старый звук звонка на 717 и еж3 тоже уместно было бы добавить.

**Comment by fixinit75** [comment]
![image](https://user-images.githubusercontent.com/78933401/111276126-0a175280-8648-11eb-969d-17c4a9093027.png)

**Comment by glebqip** [comment]
Для звонка нужен норм звук

**Comment by fixinit75** [comment]
> Для звонка нужен норм звук

Это надо у чехов просить, у них он остался.

**Comment by cyber-dream** [comment]
Они на гите есть?

**Comment by fixinit75** [comment]
Не понял немного вопрос.

**Comment by cyber-dream** [comment]
Есть ли эти чехи на гите?)

**Comment by fixinit75** [comment]
Чехов на гитхабе нет, я попросил своего товарища связаться с одним из действующих машиков там. Мб что и получим.

**Comment by fixinit75** [comment]
Так, со звонком пока что запар, чехи мне ответили, что пока Кононов ручная ситуация не нормализовалась, они не выпускают состав на линию, даже вообще не лезут в него, так что пока только звук в плохом качестве, который не стоит рассматривать пока что.

**Comment by toxin1pilot** [comment]
Звонок лежит на балконе, не проблема записать с него звук. Ем508Т головным быть не может в текущих реалиях, их штучно оборудовали в качестве головных на короткие промежутки времени и достоверной информации по из начинке нет в помине. По поводу промежуточных Еж3 - в этом нет ни малейшей необходимости

**Comment by old702** [comment]
Пиши звук. Ем508 не должен быть головным и имеется в виду Еж3 как в харькове. Проще всего решить это добавлением еж3 вагона вцепленного но по отдельной энтити

**Comment by toxin1pilot** [comment]
Звонок у меня умер, не запитывается :(

**Reply by fixinit75** [reply]
F
